js/script.js

```diff
@@ -19,10 +19,7 @@ fetch(URL).then(response => {
     ${res.city.name}, ${res.city.country}`;
     //day
     let forecast = res.list;
```

Comment:
This is a personal choice, but do you have a policy on using `let` vs `const`? Some people have a [policy](https://softwareengineering.stackexchange.com/questions/278652/how-much-should-i-be-using-let-vs-const-in-es6) to always use `const` unless they plan to reassign a variable, to avoid mistakes (although some people are dead against that)

Reply:
I only use const for what I considered immutable values, like the API URL, I was under the impression that was the correct way. Thanks for pointing this out, I just read [this](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/const) (properly this time!), as well as the stackoverflow post you sent. I'm so used to writing let for everything, but almost all of them should be const, will take a bit of time to change that habit! Thanks :)